app.js

```diff
@@ -31,7 +34,9 @@ function sendSms(caller, twilioNumber) {
         console.log("Uh oh, looks like this caller can't receive SMS messages.")
       }
     })
-    .done()
+    .done();
 }
 
 app.listen(8000);
```

Comment:
Can we add a callback here where we log out that the server is listening on port 8000? There's no feedback otherwise

Reply:
@dkundel, the callback was added. Thanks!